doc/sdk-cli-reference.md

```diff
@@ -449,6 +449,7 @@ Run scorecard tests on an operator
 * `proxy-image` string - Image name for scorecard proxy (default "quay.io/operator-framework/scorecard-proxy")
 * `proxy-pull-policy` string - Pull policy for scorecard proxy image (default "Always")
 * `-h, --help` - help for scorecard
+* `version` string - The scorecard version to run (default v1alpha1), the tech preview version is v1alpha2.
```

Comment:
@dmesser suggested that we should change the default to `v1alpha2`, and force users to opt-in to the old output version. CVP has pinned to a specific version of the SDK, so we can coordinate with them when they decide to upgrade to a newer version of the SDK containing these changes.

This should be documented as a breaking change.

Reply:
I've added a change to make v1alpha2 the default and documented it in the CHANGELOG as a breaking change.